aboutuspage.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>replit</title>
+  <link href="style.css" rel="stylesheet" type="text/css" />
+  <title> chirp website aboutus</title>
+</head>
+  
+
+<body>
```

Comment:
Awesome Work! Looks pretty good in terms of code! :)

Change Requested:
We're using React (.jsx) files in the main frontend code so a lot of this html and CSS can be transferred to the React files and prevent people from having to initiialize the <head> tags on every file. 

How To Implement Change:
Under the Pages folder, create another folder for AboutUs and create a .jsx file and a .module.css file and move all the code you in the <body> tag of this file into the .jsx file. Same thing with the .module.css file.  Checkout the Example.jsx file if you need help structuring out the .jsx file.

Reasoning:
React files are alot more flexible in terms of being able to implement functionality within the same files where you're inputting html structures. Essentially, React is a framework that combines everything you would put in a CSS, HTML, and JS file. :)